tasks/coverage/parser_babel.snap

```diff
@@ -177,6 +177,44 @@ Expect to Parse: "typescript/class/constructor-with-modifier-names/input.ts"
    ·        ╰── it cannot be redeclared here
  4 │ }
    ╰────
+Expect to Parse: "typescript/class/declare/input.ts"
```

Comment:
hmmm ... why is this failling to parse

Reply:
Babel doesn't cover all typescript errors.